### PR TITLE
fix(scrub nemesis): switch to use nodetool scrub from API

### PR DESCRIPTION
**Pull request by amoskong**
The sub-command `scrub` had been implemented.

Signed-off-by: Amos Kong <amos@scylladb.com>

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -2191,10 +2191,7 @@ def disable_disrupt_corrupt_then_scrub(self):
 
         self.log.debug("Rebuild sstable file by scrub, corrupted data file will be skipped.")
         for ks_cf in self.cluster.get_non_system_ks_cf_list(self.target_node):
-            scrub_cmd = 'curl -s -X GET --header "Content-Type: application/json" --header ' \
-                        '"Accept: application/json" "http://127.0.0.1:10000/storage_service/keyspace_scrub/{}?skip_corrupted=true"'.format(
-                            ks_cf.split('.')[0])
-            self.target_node.remoter.run(scrub_cmd)
+            self.target_node.remoter.nodetool(f"scrub {ks_cf.split('.')[0]}")
```
Comment (bentsi) [inline sdcm/nemesis.py:2194]: @amoskong since when do we have `nodetool` method on remoter?

**Review by bentsi — CHANGES_REQUESTED** [review]

File: sdcm/nemesis.py (hunk 1 above)
Comment (amoskong) [reply sdcm/nemesis.py]: Fixed.

**Comment by amoskong** [comment]
Tested the new PR by https://jenkins.scylladb.com/job/scylla-staging/job/amos/job/byo-longevity-test/44/
- test id: 13bd445d-b854-4e62-a21e-63e908e88291
- executed nemesis: CorruptThenScrubMonkey (1, succeeded)

**Review by roydahan — APPROVED** [review]

**Review by roydahan — CHANGES_REQUESTED** [review]
@amoskong although you changed it, it's till doesn't affect anything because the nemesis is disabled (starts with "disable_disrupt).
So we can maybe merge it, but we need to solve the root cause of this.

**Comment by amoskong** [comment]
> @amoskong although you changed it, it's till doesn't affect anything because the nemesis is disabled (starts with "disable_disrupt).
> So we can maybe merge it, but we need to solve the root cause of this.

This problem is tracked by other tickets:

- https://trello.com/c/vwedwZK2/1881-corrupt-the-scrub-failure
- https://trello.com/c/JTlE1RYu/1687-nodetool-scrub-implement-validation-and-the-skip-corrupted-flag

I have some progress of them but it's not solved finally. So I want to send this PR firstly to solve https://trello.com/c/jBSfwaMZ/1817-switch-to-use-nodetool-scrub-from-api-sct-dtest

Otherwise, it seems no progress of scrub tasks from PR review.

**Comment by roydahan** [comment]
ok.

**Review by roydahan — APPROVED** [review]

**Pull request merged**